### PR TITLE
feat:support push event type webhook

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/gitlab/helper.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
+package gitlab
+
+type MergeCommentEvent struct {
```
Comment (supereagle) [inline pkg/util/gitlab/helper.go]: Why need to define this?

Comment (zhujian7) [reply pkg/util/gitlab/helper.go]: https://github.com/zhujian7/cyclone/blob/bb7128664e20de4e6e0aa307620f93145e82c3d1/pkg/util/gitlab/gitlab.go#L27-L30

**Pull request merged**